fix: Added sample configuration for Microsoft Entra ID (#65)
postalserver/docs: content/3.features/oidc.md

  secret: # your client secret from Google
  scopes: [openid, email]
```

> ## Using Microsoft Entra ID as an identity provider
> 
> Setting up Postal to authenticate with Entra ID (former Azure AD) is fairly straight forward. You'll need to use the Entra ID admin center for creating an app registration, which will get you an client ID and client secret ([see docs](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app)). When prompted for a redirect URI, you should be `https://postal.yourdomain.com/auth/oidc/callback`. The following configuration can be used to enable this:

Reviewer: ```suggestion
## Using Microsoft Entra ID (formerly Azure AD) as an identity provider

You'll need to use the Entra ID admin center for creating an app registration, which will get you a client ID and client secret ([see docs](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app)). The redirect URI is `https://postal.yourdomain.com/auth/oidc/callback`. You can then complete the configuration in Postal as follows:
```
